src/fetchWebSetup.ts

```diff
@@ -12,6 +12,9 @@ export interface WebConfig {
   messagingSenderId?: string;
 }
 
+/**
+ * TODO: deprecate this function in favor of `getAppConfig()` in `/src/management/apps.ts`
+ */
 export async function fetchWebSetup(options: any): Promise<WebConfig> {
```

Comment:
@mbleigh, @bkendall, @nbegley   I am reluctant to remove this method since it's used in `implicitInit.js` which is used in `emulators:exec` and `emulators:start` commands. It's due to the fact that `fetchWebSetup` requires `projectId`, while `getAppConfig` requires `appId` instead.
We need to resolve this related conversation in the design proposal as well: https://docs.google.com/a/google.com/document/d/1AmJ6_ALtcsdz27zQt71Zu699uuQ6NL2OCMHYtWDhvAI/edit?disco=AAAADYsQdmU

Reply:
@samtstern @abeisgoat - any thoughts here?

Reply:
Leave the comment here to have it be deprecated. I'll follow up with folks out-of-band.

Reply:
Thanks